src/main/resources/application.yaml

```diff
@@ -68,7 +68,7 @@ idam:
     client-authorization: ${OAUTH2_CLIENT_AUTH:cmQtcHJvZmVzc2lvbmFsLWFwaTpjYzVmMmE2LTk2OTAtMTFlOS1iYzQyLTUyNmFmNzc2NGY2NA==}
 
```

Comment:
Check if below hardcoded secrets can be replaced with dummy values as actual values will come from vault 
authorization: ${OAUTH2_AUTH
client-authorization: ${OAUTH2_CLIENT_AUTH

Reply:
done